src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/UserServiceTest.kt

```diff
@@ -564,11 +583,7 @@ class UserServiceTest {
 
       var entity = result.entity
 
-      assertThat(entity.id).isEqualTo(user.id)
-      assertThat(entity.name).isEqualTo("$forename $surname")
-      assertThat(entity.deliusUsername).isEqualTo(user.deliusUsername)
       assertThat(entity.email).isEqualTo("null")
-      assertThat(entity.telephoneNumber).isEqualTo(deliusUser.telephoneNumber)
```

Comment:
As this test describes itself as testing the email and all the other fields are tested in `it returns the user's details from the Community API and saves the email address, telephone, staff code and probation region` this seemed redundant. 

It also seems like the checks have diverged with the other test checking for deliusStaffCode where this one isn't. Let's keep this test focused around email?

Reply:
Sounds sensible 👍